tests/foreman/api/test_http_proxy.py

```diff
@@ -293,7 +293,7 @@ def test_positive_sync_proxy_with_certificate(request, target_sat, module_org, m
     # Create and fetch new cerfiticate
     target_sat.custom_cert_generate(proxy_host)
     cacert = target_sat.execute(f'cat {cacert_path}').stdout
-    assert 'BEGIN CERTIFICATE' and 'END CERTIFICATE' in cacert
+    assert 'END CERTIFICATE' in cacert
```

Comment:
We're missing assertion for 'BEGIN CERTIFICATE' here!

Reply:
That shouldn't be a concern since the master branch contains an identical patch. See [this patch](https://github.com/SatelliteQE/robottelo/pull/12796/files#diff-878c1bd8e5143731475760ea2f4334b26db44f14b306b3c5fbebb4f7a1d63c11L296-R296).

Reply:
Correct, this was a decision from the test owner.

Reply:
Cool, it's good to have but as @damoore044 confirmed this, so I wouldn't block on this